test/integrations/destinations/smartly/processor/track.ts

```diff
@@ -0,0 +1,71 @@
+import { destination } from '../commonConfig';
+
+export const trackTestData = [
+  {
+    name: 'smartly',
+    description: 'Test 0',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            destination,
+            message: {
+              event: 'Add to cart',
+              properties: {
+                platform: 'meta',
+                ad_unit_id: '228287',
+                ad_interaction_time: 1735680000,
+                email: 'eventIdn01@sample.com',
+              },
+              type: 'track',
+              userId: 'eventIdn01',
+            },
+            metadata: {
+              destinationId: 'dummyDestId',
+              jobId: '1',
+            },
+          },
+        ],
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            metadata: {
+              destinationId: 'dummyDestId',
+              jobId: '1',
+            },
+            output: {
+              version: '1',
+              type: 'REST',
+              method: 'POST',
+              endpoint: 'https://s2s.smartly.io/events/batch',
```

Comment:
@aanshi07 we are using batch endpoint here but json payload is that of single event endpoint

Reply:
Right @anantjain45823 , I have added the changes as It won't be a problem adding single event endpoint here.